packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -11,7 +11,7 @@ export interface FinalFormSelectProps<T> extends FieldRenderProps<T, HTMLInputEl
     children?: React.ReactNode;
 }
 
-export const FinalFormSelect = <T extends Record<string, any>>({
+export const FinalFormSelect = <T,>({
```

Comment:
what is this extra `,` good for?

Reply:
This is related to arrow functions and JSX. If there's no comma, the <T> is interpreted as opening tag:

<img width="821" alt="image" src="https://user-images.githubusercontent.com/48853629/168968433-937cc923-5348-43a7-b3a8-18e77ca9ec56.png">

This is because `const FinalFormSelect = <T>...</T>` would be a valid React component 😂